### PR TITLE
Added folder structure, Jenkins file and Docker file for Housekeeping framework

### PR DESCRIPTION
**Pull request by souravbhowmik21**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        always {
+            sendSlackNotification currentBuild.result
+        }
+      }
+    }
+  }
+  environment {
+    APPLICATION = 'autoniq-etl-housekeeper'
+    APPLICATION_NAME = 'Autoniq ETL Housekeeper'
+    IMAGE_URL = '055676811907.dkr.ecr.us-east-2.amazonaws.com'
```
Comment (bplabombarda-drivin) [inline Jenkinsfile]: Jenkins-tools requires a slash adter the `IMAGE_URL`. Please add here and remove the slash after the `IMAGE_URL` variable within shell scripts in this file. 

E.g. all instances of `$IMAGE_URL/$APPLICATION` should become `$IMAGE_URL$APPLICATION`

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  }
+  stages {
+    stage('Dockerize') {
+      when {
+        beforeAgent true
+        branch 'development'
+      }
+      steps {
+        script {
+          def date = new Date()
```
Comment (bplabombarda-drivin) [inline Jenkinsfile]: Change 19-20 to use the singular date method. See the [VERO Downloader Jenkinsfile](https://github.com/sweetride/autoniq-etl-vero-downloader/blob/development/Jenkinsfile#L19).

File: utils/source_manager.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import os
```
Comment (bplabombarda-drivin) [inline utils/source_manager.py]: Nit: Not sure if we need this file, but we can cleanup out utils later.

**Comment by bplabombarda-drivin** [comment]
Also will need a task def template in the project root. The name must be `[repo_name]-definition.json`. You can copy the [VERO Downloader task def template](https://github.com/sweetride/autoniq-etl-vero-downloader/blob/development/autoniq-etl-vero-downloader-definition.json) verbatim.

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+      when {
+        beforeAgent true
+        branch 'development'
+      }
+      environment {
+        ENV='dev'
+      }
+      steps {
+        echo 'Deploying to Dev'
+        sh '''
```
[the remark this answers is not in the document]
Comment (souravbhowmik21) [reply Jenkinsfile]: Yes, this is a time scheduled task. To be run once a day

[the remark this answers is not in the document]
Comment (bplabombarda-drivin) [reply Jenkinsfile]: Gotcha - all the comments here assume that, so they should all be accurate 😄

**Comment by souravbhowmik21** [comment]
Just pushed with the new changes

**Pull request merged**